repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/container/container-configuration.xsd

```diff
@@ -149,6 +218,20 @@
         </xs:simpleContent>
     </xs:complexType>
 
+    <xs:complexType name="ArtifactDirectoryPatch">
```

Comment:
Oops. I removed the other, but apparently missed this one. I'll remove it for now.